Realm/ObjectServerTests/SwiftFlexibleSyncServerTests.swift

```diff
@@ -1186,6 +1186,30 @@ class SwiftFlexibleSyncTests: SwiftSyncTestCase {
         XCTAssertEqual(downloadCount.value, 0)
         XCTAssertEqual(uploadCount.value, 0)
     }
+
+    func testFlexibleSyncNotEnabledError() throws {
+        let appServerId = try RealmServer.shared.retrieveAppServerId(appId)
```

Comment:
Creating a new app with sync disabled is probably faster than toggling sync off and on and eliminates the chance of something going wrong in this test causing problems for other tests. This could be done by adding a new value to `SyncMode` and exiting createApp() after setting up the auth providers.